.gitlab-ci.yml

```diff
@@ -0,0 +1,72 @@
+stages:
+  - build
+  - test
+  - report
+
+before_script:
+  - sh -c 'echo "deb [arch=amd64,arm64] http://repo.ros2.org/ubuntu/main `lsb_release -cs` main" > /etc/apt/sources.list.d/ros2-latest.list'
+  - curl http://repo.ros2.org/repos.key | apt-key add -
+  - apt-get update
+  - apt-get install -y python3-colcon-common-extensions
+  - source /opt/ros/crystal/setup.bash
+
+build_launchtest:
+  stage: build
+  image: osrf/ros2:nightly
```

Comment:
I'd prefer to have this run on osrf/ros2:bouncy or osrf/ros2:crystal but it doesn't look like those docker images are available anymore.  I'm guessing they were in the past, because ApexAI [performance test package](https://github.com/ApexAI/performance_test/blob/master/.gitlab-ci.yml#L7) looks like it used it before

Reply:
Try FROM osrf/ros:crystal-desktop

Reply:
This works, except that we depend on a slightly newer version of launch than what's in crystal.  For now, I'll leave this on the 'nightly'